test/integration/test_konnect_entities.go

```diff
@@ -143,6 +143,22 @@ func TestKonnectEntities(t *testing.T) {
 		assertKonnectEntityProgrammed(t, kpb)
 	}, testutils.ObjectUpdateTimeout, testutils.ObjectUpdateTick)
 
+	globalKPB := deploy.KongPluginBinding(t, ctx, clientNamespaced,
+		konnect.NewKongPluginBindingBuilder().
+			WithPluginRef(kp.Name).
+			WithControlPlaneRefKonnectNamespaced(cp.Name).
+			WithScope(configurationv1alpha1.KongPluginBindingScopeGlobalInControlPlane).
+			Build(),
+		deploy.WithTestIDLabel(testID),
+	)
+
+	t.Logf("Waiting for KongPluginBinding to be updated with Konnect ID")
+	require.EventuallyWithT(t, func(t *assert.CollectT) {
+		err := GetClients().MgrClient.Get(GetCtx(), types.NamespacedName{Name: globalKPB.Name, Namespace: globalKPB.Namespace}, globalKPB)
+		require.NoError(t, err)
```

Comment:
And probably it makes sense to do
```suggestion
		if if !assert.NoError(t, err) {
				return
		}
```

Reply:
Double `if` suggested 🙃

Reply:
As explained in https://github.com/Kong/gateway-operator/pull/1052#discussion_r1923536941.

Reply:
Oh, good to know 👍